dev/com.ibm.ws.webcontainer.servlet.3.1_fat.2/test-applications/LibertyReadListenerFilterTest.war/resources/WEB-INF/web.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright (c) 2014 IBM Corporation and others.
```

Comment:
update cc

Reply:
File existed here :https://github.com/OpenLiberty/open-liberty/blob/integration/dev/com.ibm.ws.webcontainer.servlet.3.1_fat/test-applications/LibertyReadListenerFilterTest.war/resources/WEB-INF/web.xml I think git had issues figuring out all the renames and moves in this PR